Started adding instructions to the shell script files. (#16)
cmelab/planckton-flow: README.md

* Fry
* Kestrel

> ### .sh file instructions

Reviewer: ```suggestion
<details>
     <summary>## Included shell scripts (Click to expand)</summary>
```
I suggest that we keep this heading on the same level as "Cluster support" so that it doesn't seem like a subheading to the previous one. Also I changed the wording to (1) start the heading with a capital letter and (2) give an explanation of what they are vs what extension they have. "Instructions for included shell scripts" could also work but is a little wordy... 🤔 

I think I also like the idea of moving the collapsible part here?

> <details>
>      <summary>Clean.sh (Click to expand):</summary>

Reviewer: ```suggestion
    - clean.sh
```
we could use numbering or bullets here to keep the instructions for the two scripts distinct

>      Clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, within the present planckton-flow clone you are using. To run this shell script:

Reviewer: ```suggestion
     clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
```
Added a little more detail

>     ```bash
>     bash clean.sh 
>      ```

Reviewer: ```suggestion

    ```bash
    bash clean.sh 
    ```

```
I think these aren't rendering because they need blank lines around them

>      in your command line.**MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON_FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM.

Reviewer: ```suggestion
     in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
```
fix bold rendering and added more warning

>      <summary>Print_status.sh (Click to expand):</summary>

Reviewer: ```suggestion
     - print_status.sh
```
if you choose the suggestions above, this is more in line with those

>      Print_status.sh is a shell script file that prints out the status of your recently submitted jobs. To run this shell script:

Reviewer: ```suggestion
     print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:
```

>      ```bash
>      bash print_status.sh
>       ```

Reviewer: ```suggestion

     ```bash
     bash print_status.sh
     ```

```

Author: Awesome, thanks for the feedback and additions!

* Fry
* Kestrel

> ## Included shell scripts
> <details>
>      <summary>clean.sh</summary>
> `clean.sh` is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching `status.txt`, `signac*`, `*.log`, `*.gsd`, or `*.out` within the present planckton-flow clone you are using. To run, type the following in your terminal:
> 
>     bash clean.sh 
> 
> **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
> </details>
> <details>
>      <summary>print_status.sh</summary>
> `print_status.sh` is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run, type the following in your terminal:
> 
>      bash print_status.sh
> 
> </details>

Reviewer: ```suggestion
## Included shell scripts
<details>
     <summary>clean.sh</summary>
clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run, type the following in your terminal:

    bash clean.sh 

**THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
<details>
     <summary>print_status.sh</summary>
print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run, type the following in your terminal:

     bash print_status.sh

</details>
```

* Fry
* Kestrel

## Included shell scripts

> <details>
>      <summary>clean.sh (Click to expand)</summary>
> clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
> 
>     bash clean.sh 
> 
> in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
> </details>

Reviewer: ```suggestion
<details>
     <summary>clean.sh</summary>
`clean.sh` is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching `status.txt`, `signac*`, `*.log`, `*.gsd`, or `*.out` within the present planckton-flow clone you are using. To run, type the following in your terminal:

    bash clean.sh 

**THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
```

<details>

>      <summary>`print_status.sh` (Click to expand)</summary>
> print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:
> 
>      bash print_status.sh
> 
> in your command line.

Reviewer: ```suggestion
     <summary>print_status.sh</summary>
`print_status.sh` is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run, type the following in your terminal:

     bash print_status.sh

```

* Fry
* Kestrel

> <details>
>      <summary>## Included shell scripts (Click to expand)</summary>

Reviewer: ```suggestion
## Included shell scripts
```
The formatting here is a little funky, so let's try it back to the way you originally had it!

>     - clean.sh
>      clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
> 
>     ```bash
>     bash clean.sh 
>     ```
> 
>      in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**

Reviewer: ```suggestion
<details>
     <summary>clean.sh (Click to expand)</summary>
clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:

    bash clean.sh 

in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
```
trying to fix formatting... things behave strangely with the `<details>` tags

>   <details>
>      - print_status.sh
>      print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:
> 
>      ```bash
>      bash print_status.sh
>      ```
> 
>       in your command line.

Reviewer: ```suggestion
<details>
     <summary>`print_status.sh` (Click to expand)</summary>
print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:

     bash print_status.sh

in your command line.
</details>
```
